lib/hllcounter.cc

```diff
@@ -433,16 +376,15 @@ void HLLCounter::consume_fasta(
                 }
 
                 #pragma omp task default(none) firstprivate(read) \
-                shared(counters, n_consumed_partial, total_reads_partial)
+                shared(hlls, n_consumed_partial, total_reads_partial)
                 {
-                    bool is_valid;
                     int n, t = omp_get_thread_num();
-                    n = counters[t]->check_and_process_read(read.sequence,
-                                                            is_valid);
+
+                    read.set_clean_seq();
```

Comment:
I still don't like that we have to remember to call this. Can we move this to `FastxReader::get_next_read`? Would mean it is always done for everyone, pro: can't forget it, con: extra memory used for everyone.